cylc/flow/network/schema.py

```diff
@@ -1072,6 +1078,11 @@ class Meta:
     depth = Int(
         description='The family inheritance depth',
     )
+    graph_depth = Int(
+        description=sstrip('''
+            The n-window graph edge depth from closet active task(s).
+        '''),
+    )
```

Comment:
Because this is the depth of the task proxy, the other is the max graph depth (the n in n-window).

Reply:
(this is the field that the cylc-ui diff is using to colour in tasks by n-window depth, the other field will be used by the ui to display the configured n-window depth so that the user can change the window they are viewing)